### ISSUE TITLE
Proposals around a better use of ISCED-F 2013 

### ISSUE THREAD

**Issue by init-xbildung** [emrex-eu/elmo-schemas#60]
In ELMO disciplines can be expressed using ISCED-F 2013.
```
 <xs:element minOccurs="0" name="iscedCode" type="xs:token">
                <xs:annotation>
                    <xs:documentation>The ISCED-F code of the course or programme.

                        ISCED-F codes define fields of education and training at the secondary,
                        post-secondary and tertiary levels of education. Details:
                        http://www.uis.unesco.org/Education/Documents/isced-fields-of-education-training-2013.pdf
                    </xs:documentation>
                </xs:annotation>
```
However some disciplines are not further defined by ISCED-F2013 as other vocabularies exist for that purpose. This is especially true languages.

For languages:

In the German XSchule project we try to express all secondary education disciplines in a machine readable way by 
complementing the ISCED-F 2013 with the ISO 3166-2 language code list to express languages as a discipline:

See: https://xschule.digital/web/kmk_abiturfaecher_de for a complete list of all disciplines and its mapping to ISCED-F 2013 and ISO 3166-2

Proposal No.1 is to extend the possibility of indicating a discipline with ISCED-F 2013 with the language code list.

In doing so digital job applications with ELMO credentials might be filtered by employer side with
"_finding people having had a good grade in English and French"_ instead of filtering for people 
"_having had a good grade in Language Acquisition"_ (this is where ISCED-F 2013 stops - not sure whether languageOfInstruction will do the trick)

Proposal No2: as there is also the ISCED-2011 Level, the name of the ISCED-F 2013 property in ELMO could be more explicit than `ISCEDCode` to achieve forward-compatibility

Proposal No3: When using ISCED2013 use URIs instead of strings:

As said for the country code list in https://github.com/emrex-eu/elmo-schemas/issues/57 Multilinguality support could be enhanced using the Publications Office URIs instead of the string values of ISCED2013.

Example used in ELMO https://github.com/emrex-eu/elmo-schemas/blob/v1/example.xml
```
<type>Degree Programme</type>
<iscedCode>0511</iscedCode>
<url>http://www.uio.no/studier/program/biologi-master/</url>
```
could better be
```
<type>Degree Programme</type>
<iscedF2013Code>https://publications.europa.eu/resource/authority/snb/isced-f/0511</iscedCode>
<url>http://www.uio.no/studier/program/biologi-master/</url>
```
to benefit from the multilingual resources of DG Translation and the Publications Office.
![image](https://user-images.githubusercontent.com/99474274/169414149-19255d02-227a-45a5-80b4-2b2ae8abc9b3.png)

**Comment by mirkostanic** [comment]
To be implemented in v2.0

**Comment by mirkostanic** [comment]
Additional granularity can be achieved through the use of newly added gradingSchemeExternalId element.
Using URIs is subject to link rot.
